clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -47,3 +48,8 @@ spec:
             storage: {{MultiplyInt 10 (AddInt 1 (DivideInt .Nodes 1000))}}Gi
   query:
     maxSamples: 100000000
+  {{if $PROMETHEUS_SCRAPE_WINDOWS_NODES}}
+  additionalScrapeConfigs:
```

Comment:
I think you could move additionalScrapeConfigs outside the if block

Reply:
Do you mean moving the additionalScrapeConfigs to the top of if block? I tried but got this error if `$PROMETHEUS_SCRAPE_WINDOWS_NODES` is false.
```
spec.additionalScrapeConfigs: Invalid value: "null": spec.additionalScrapeConfigs in body must be of type object: "null"
```

Reply:
Thanks for checking! Let's leave it as is in this case.